package.json

```diff
@@ -5,9 +5,22 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "scripts": {
-    "build": "tsc",
-    "watch": "tsc -w",
-    "prepublish": "npm run build"
+    "build": "jsii",
+    "watch": "jsii -w",
+    "prepublish": "npm run build",
+    "build:bindings": "jsii-pacmak"
```

Comment:
This should be called "package" per our conventions

Reply:
I might want to revisit those conventions (to be more node-idiomatic in certain ways). But regardless... Consistency is key 👌🏻